README.md

```diff
@@ -1,88 +1,97 @@
-# TrueBlocks for dAppNode
+# TrueBlocks on Docker
 
 ![Image Logo](https://avatars1.githubusercontent.com/u/19167586?s=200&v=4)
 
 [![Website trueblocks.io](https://img.shields.io/badge/Website-quickblocks.io-brightgreen.svg)](https://quickblocks.io/)
 ![Docker](https://img.shields.io/badge/trueblocks-docker-blue.svg)
 [![Twitter](https://img.shields.io/twitter/follow/espadrine.svg?style=social&label=Twitter)](https://twitter.com/quickblocks?lang=es)
 
-The dAppNode project answers the "how" question for running an Ethereum node. TrueBlocks answers the "why" question. dAppNode makes setting up and running a node easier. TrueBlocks makes extracting useful data ***from your particular list of accounts*** from that node easier.
+TrueBlocks extracts address appearances from an Ethereum chain and creates a lightweight index, making queries by address a trustless and fast possibility.
 
-## Requirements
+trueblocks-docker is a docker image containing an installation of TrueBlocks, and an http API server that delivers Ethereum data by address.
 
-- A running Parity Ethereum node with `--tracing on` enabled. You can do this on the  [DNP_ETHCHAIN](https://github.com/dappnode/DNP_ETHCHAIN) package using `EXTRA_OPTS=--tracing on` ([Why do we need --tracing enabled?]())
-- An additional 100GB of hard drive space for the TrueBlocks caches.
-- Time (it takes a long time to sync the TrueBlocks caches: [why?]()).
+<!-- By visiting every bit of the Ethereum data including blocks, transactions, receipts, logs, traces, trace actions, and trace results for every block, TrueBlocks is able to extract every 'appearance' of an address on the blockchain.
 
-## Usage examples
+Trueblocks finds nearly 30% more appearances than naive approaches such as EtherScan. Sorting the list and creating an index allows TrueBlocks to reconstruct historical lists of transactions per address more than 150,000 times faster than a raw scan of the chain via the RPC.
```

Comment:
Maybe we can say something like "...than other more naive approaches" and not mention anyone.

Reply:
I agree that calling out etherscan here doesn't seem productive. This section is currently commented out. Let's come back to this on the next iteration.